### PR TITLE
Add linux support.

### PR DESCRIPTION
**Pull request by feluxe**
We are a mixed team of Mac/Linux users, therefor Linux support would be quite helpful for us. :)

### CODE REVIEW

File: kn.rb [hunk 1]
```diff
@@ -1,19 +1,34 @@
-require "FileUtils"
+if OS.mac?
+  require "FileUtils"
+else
```
Comment (mattmoor) [inline kn.rb:3]: Remove trailing whitespace:
```suggestion
else
```

File: kn.rb [hunk 2]
```diff
@@ -1,19 +1,34 @@
...
 
   v = "v0.15.1"
-  url "https://github.com/knative/client/releases/download/#{v}/kn-darwin-amd64"
   version v
-  sha256 "b594731a04f49c6b130245193aaf93eb53235ca01abed9b5a97fb54ef8c18577"
+
+  if OS.mac?
+    url "https://github.com/knative/client/releases/download/#{v}/kn-darwin-amd64"
+    sha256 "b594731a04f49c6b130245193aaf93eb53235ca01abed9b5a97fb54ef8c18577"
+  else
```
Comment (mattmoor) [inline kn.rb:16]: Remove trailing whitespace:
```suggestion
  else
```

File: kn.rb [hunk 3]
```diff
@@ -1,19 +1,34 @@
...
+  else 
+    url "https://github.com/knative/client/releases/download/#{v}/kn-linux-amd64"
+    sha256 "aa66d8f3c442219522bc8a135aa04b25700e2da9153b436c2d8259c6705ff88d"
+  end
 
   def install
-    FileUtils.mv("kn-darwin-amd64", "kn")
+    if OS.mac?
+      FileUtils.mv("kn-darwin-amd64", "kn")
+    else
```
Comment (mattmoor) [inline kn.rb:24]: Remove trailing whitespace:
```suggestion
    else
```

**Comment by feluxe** [comment]
@googlebot I signed it!

**Comment by rhuss** [comment]
Thanks ! 

/lgtm
/approve

**Pull request merged**